include/rocksdb/cloud/cloud_file_system.h

```diff
@@ -628,6 +631,16 @@ class CloudFileSystem : public FileSystem {
                                     std::vector<std::string>* live_sst_files,
                                     std::string* manifest_file) = 0;
 
+  // Read the given manifest file (either the one used in the database or a
+  // copy) and populate live_sst_files with the list of all SST files that it
+  //
+  // references. Unlike FindAllLiveFiles, this method doesn't resolve the
+  // manifest name from the cloud manifest and also doesn't pull the manifest
+  // (i.e., it needs to be present locally).
+  virtual IOStatus ReadManifestLiveFiles(
```

Comment:
can we name it "FindLiveFilesFromLocalManifest()" so that it is similar naming to existing method names?